kolibri/auth/filters.py

```diff
@@ -0,0 +1,185 @@
+from django.db import models
+from django.db.models.query import F
+from six import string_types
+
+from .constants import collection_kinds
+
+
+class HierarchyRelationsFilter(object):
```

Comment:
> If you think that's easy to set up, let's do it (I'd need direction). Otherwise, I'd say let's cross that bridge when we get there.

I do think it would be fairly straightforward to set up:
1. Get postgres running in our travis environment: https://docs.travis-ci.com/user/database-setup/#PostgreSQL
2. Create a new settings module that uses a postgres test database: https://docs.djangoproject.com/en/1.9/topics/testing/overview/#the-test-database
3. Add a new tox configuration to run the tests using that settings module.

If supporting postgres is important to you, then we're at the bridge IMO. So I'm going to set it up right now.